pkg/logger/log/accesslog.go

```diff
@@ -226,17 +226,18 @@ func AccessLog(req *restful.Request, resp *restful.Response, chain *restful.Filt
 
 // getRequestBody will get the request body from Request object
 func getRequestBody(req *restful.Request, contentType, requestURL string) (string, float32) {
-	if contentType == "" || !isSupportedContentType(contentType) {
+	if contentType == "" || !isSupportedContentType(contentType) || req.Request == nil || req.Request.Body == nil {
 		return "", 0
 	}
 
-	bodyBytes, err := ioutil.ReadAll(req.Request.Body)
+	bodyBytes, err := io.ReadAll(req.Request.Body)
```

Comment:
we will set the original bytes back to req.Request.Body, so no need to close it here.

Reply:
I think we should not `return` here immediately (fail fast),
the reason, if we got `err`, there's a change the `io.ReadAll` already read the bytes of `req.Request.Body` reader, 
So we need to set it back to the `req.Request.Body` in the line 240.
If we're not set it back, the main API function that will got error later, because it expect to read the `req.Request.Body` Reader, but it's empty (because it have been read in this code).

In this case, we need continue to the below logic and no need to fail fast.